release/pipline/operators/gcs_hook.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from apiclient.discovery import build
+from apiclient.http import MediaFileUpload
+from googleapiclient import errors
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class GoogleCloudStorageHook(GoogleCloudBaseHook):
+    """
+    Interact with Google Cloud Storage. This hook uses the Google Cloud Platform
+    connection.
+    """
+
+    def __init__(self,
+                 google_cloud_storage_conn_id='google_cloud_storage_default',
+                 delegate_to=None):
+        super(GoogleCloudStorageHook, self).__init__(google_cloud_storage_conn_id,
+                                                     delegate_to)
+
+    def get_conn(self):
+        """
+        Returns a Google Cloud Storage service object.
+        """
+        http_authorized = self._authorize()
+        return build('storage', 'v1', http=http_authorized)
+
+    # pylint:disable=redefined-builtin
+    def copy(self, source_bucket, source_object, destination_bucket=None,
```

Comment:
A lot of lines here and in gcs_to_gcs.py (totally 770 lines) is implementing a gcs bucket to bucket copy operator.  Maybe reuse/refactor store_artifacts.sh instead?

Reply:
This file is 291. It is used because it part of airflow. Our version is behind master and this feature has not been fully merged into the release branch. Once it is our these files will not be needed.